src/components/ReportActionItem/IOUPreview.js

```diff
@@ -139,10 +139,10 @@ const IOUPreview = (props) => {
 
     const moneyRequestAction = ReportUtils.getMoneyRequestAction(props.action);
 
-    const requestAmount = moneyRequestAction.amount;
-    const requestCurrency = moneyRequestAction.currency;
-    const requestComment = Str.htmlDecode(moneyRequestAction.comment).trim();
-
+    // If props.action is undefined then we are displaying within IOUDetailsModal and should use the full report amount
+    const requestAmount = props.isIOUAction ? moneyRequestAction.amount : ReportUtils.getMoneyRequestTotal(props.iouReport);
+    const requestCurrency = props.isIOUAction ? moneyRequestAction.currency : props.iouReport.currency;
+    const requestComment = moneyRequestAction.comment.trim();
```

Comment:
@AngelNBazan I think we can do nothing for the requested amount and the currency since `isIOUAction` is removed. Just need to update the comment.

Reply:
Ok, we should revert the above change. The currency should be IDR not USD.

![Screenshot 2023-05-30 at 18 40 38](https://github.com/Expensify/App/assets/25520267/edb3cfad-3b33-447a-acd6-8ea570b37d7e)